core/src/main/java/org/elasticsearch/monitor/os/OsStats.java

```diff
@@ -166,22 +170,70 @@ public static Cpu readCpu(StreamInput in) throws IOException {
         @Override
         public void readFrom(StreamInput in) throws IOException {
             percent = in.readShort();
-            loadAverage = in.readDouble();
+            loadAverage = new LoadAverage();
+            loadAverage.readFrom(in);
         }
 
         @Override
         public void writeTo(StreamOutput out) throws IOException {
             out.writeShort(percent);
-            out.writeDouble(loadAverage);
+            loadAverage.writeTo(out);
         }
 
         public short getPercent() {
             return percent;
         }
 
-        public double getLoadAverage() {
+        public LoadAverage getLoadAverage() {
             return loadAverage;
         }
+
+        public static class LoadAverage implements Streamable {
+            private double oneMinuteLoadAverage;
+            private double fiveMinuteLoadAverage;
+            private double fifteenMinuteLoadAverage;
+
+            public final static LoadAverage DEFAULT_LOAD_AVERAGE = new LoadAverage(-1, -1, -1);
```

Comment:
> does this constant make sense to be here or in the fallback code should we just pass new LoadAverage(-1, -1, -1).

@rmuir I was looking to avoid creating unnecessary garbage when there is periodic monitoring in place and these statistics aren't available (specifically Windows).

> Maybe we should remove the ctor taking a single double as well, and pass new LoadAverage(x, -1, -1). I had trouble following the logic but then it would be all clear what values are being returned in those cases.

That's fine. I pushed